docs/user_guide/bank/inventory_items.md

```diff
@@ -1 +1,156 @@
-Not yet written
\ No newline at end of file
+DRAFT USER GUIDE
+# Items
+## Introduction
+Your bank is initialized with  a basic set of items that contains many common product types that essentials banks distribute.   These items represent the stock you have for distribution.  Here is the current default list:  
+
+Adult Briefs (Large/X-Large),
```

Comment:
We'll need to bulletize or do something with this list to make it not all jammed together in a paragraph in a future-pass

Reply:
Added as a todo in the set.